### PR TITLE
Update to include missed stop area updates

### PR DESCRIPTION
**Pull request by br648**
### Checklist

- [x] Appropriate branch selected _(all PRs must first be merged to `dev` before they can be merged to `master`)_
- [x] Any modified or new methods or classes have helpful JavaDoc and code is thoroughly commented
- [x] The description lists all applicable issues this PR seeks to resolve
- [x] The description lists any configuration setting(s) that differ from the default settings
- [x] All tests and CI builds passing

### Description

Update to include a couple of locations where I had missed stop area enhancements.

### CODE REVIEW

**Review by philip-cline — APPROVED** [review]
This works for me.

File: src/main/java/com/conveyal/gtfs/model/StopTime.java [hunk 1]
```diff
@@ -212,8 +212,8 @@ protected Iterator<StopTime> iterator() {
 
     /**
      * Check that the flex column 'start_pickup_drop_off_window' existing in the stop time table. If this column exists
```
Comment (binh-dam-ibigroup) [inline src/main/java/com/conveyal/gtfs/model/StopTime.java]:
```suggestion
     * Check that the flex column 'start_pickup_drop_off_window' exists in the stop time table. If this column exists
```

File: src/main/java/com/conveyal/gtfs/validator/ReferencesTripValidator.java [hunk 2]
```diff
@@ -65,7 +65,7 @@ public void validateTrip(
         });
         stopAreas.forEach(stopArea -> {
             if (stopArea != null) {
-                referencedLocations.add(stopArea.area_id);
+                referencedStopAreas.add(stopArea.area_id);
```
Comment (binh-dam-ibigroup) [inline src/main/java/com/conveyal/gtfs/validator/ReferencesTripValidator.java:68]: Did you intend to merge `referencedLocations` and `referencedStopAreas`?

**Review by binh-dam-ibigroup — COMMENTED** [review]
One variable was ambiguously renamed, could you check, please?

File: src/main/java/com/conveyal/gtfs/validator/ReferencesTripValidator.java (hunk 2 above)
Comment (br648) [reply src/main/java/com/conveyal/gtfs/validator/ReferencesTripValidator.java:68]: No, was a typo! In this case `referencedLocations` should definitely be `referencedStopAreas`.

Comment (binh-dam-ibigroup) [reply src/main/java/com/conveyal/gtfs/validator/ReferencesTripValidator.java:68]: ~Then why is `referencedLocations` still being used (e.g. line 63)~ Nvm

**Review by binh-dam-ibigroup — APPROVED** [review]
LGTM

**Pull request merged**